lisa/tools/reboot.py

```diff
@@ -55,7 +55,8 @@ def reboot_and_check_panic(self, log_path: Path) -> None:
                 raise LisaException(f"after reboot, {identifier}")
             raise identifier
 
-    def reboot(self) -> None:
+    def reboot(self, time_out: int = 300) -> None:
+        self.time_out = time_out
```

Comment:
the self.time_out is not useful anymore.

Reply:
removed it